pages/docs/sharing-data-externally/email-reports.mdx

```diff
@@ -0,0 +1,58 @@
+# E-mail Reports
+
+You can generate e-mail reports for any Saved Exploration or Dashboard in Glean.
+
+- [How to Send an E-mail Report](#how-to-send-an-e-mail-report)
+- [Setting Up E-mail Reports](#setting-up-e-mail-reports)
+- [Scheduling E-mail Reports](#scheduling-e-mail-reports)
+- [What Happens When Reports Fail](#what-happens-when-reports-fail)
+- [Managing Scheduled Reports](#managing-scheduled-reports)
+
+## How to Send an E-mail Report
```

Comment:
This and the headings below should be sentence case unless we've changed our tune since earlier this summer. Looks like we've diverged a bit still since then on this standard - I can take a pass next week to renormalize us.